MonopolyKata/Monopoly.cs

```diff
@@ -91,5 +129,64 @@ public void RunMonopoly()
             }
         }
 
+        private Int32 CalculateMultipleGroupMultiplier(String owner, Int32 currentLocation)
+        {
+            var multiplier = 1;
+            var count = 0;
+            var person = stringToPlayer[owner];
+            var type = gameBoard.GetType(currentLocation);
+            var color = gameBoard.GetColor(currentLocation);
+
+            switch (gameBoard.GetType(currentLocation))
+            {
+                case "Property":
+                    foreach (String element in person.ownedProperties)
+                    {
+                        if (person.propertyColor[element] == color)
+                            count++;
+                    }
+                    if (count == 2 && color == "Blue" || color == "Pink")
+                        multiplier = 2;
+                    else if (count == 3 && color != "Blue" && color != "Pink" && color != null)
+                        multiplier = 3;
+                    break;
+                case "Utility":
+
+                    foreach (String element in person.ownedProperties)
+                        if (person.typeOfProperty[element] == type)
+                            count++;
```

Comment:
In terms of Qualtrax style, we use brackets on blocks more than one line and no brackets on blocks that are one line.